typescript/sdk/src/router/schemas.ts

```diff
@@ -24,13 +24,19 @@ export const RemoteRoutersSchema = z.record(
   RemoteRouterRouter,
 );
 
+const ProxyAdminConfigSchema = z.object({
+  proxyAdmin: z.string().optional(),
```

Comment:
this is also an artifact of the warp route deployment so this is subtlely mixing config and artifact schemas

Reply:
The `RouterConfig` type is used for an artifact of the warp route deployment?